pkg/metrics/metrics.go

```diff
@@ -101,6 +116,29 @@ var (
 		Help:      "Number of policies currently loaded",
 	})
 
+	// PolicyRegenerationCount is the total number of policies successfully
+	// imported and generated.
```

Comment:
Remove "imported". This is not what is counted by this metric.
I'd just write:
```
// PolicyRegenerationsTotal is the total number of successful policy regenerations.
```

Reply:
done